devtools/install_requirements.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Conflict: this requires numpy<2 whereas ExecuTorch core requires numpy>=2
```

Comment:
How does this will work, once ET moves to numpy>=2?
Does Google have a plan to upgrade to a newer numpy?